src/private/meetingAudio.ts

```diff
@@ -0,0 +1,34 @@
+import { ensureInitialized, sendMessageRequestToParent } from '../internal/internalAPIs';
+import { GlobalVars } from '../internal/globalVars';
+import { SdkError, ErrorCode } from '../public/interfaces';
+
+export namespace meetingAudio {
+  /**
+   * @private
+   * Hide from docs
+   * ------
+   * Allows an app to toggle the incoming audio speaker setting for the meeting user from mute to unmute or vice-versa
+   * @param callback Callback contains 2 parameters, error of type SdkError and result of type boolen or null.
+   * result can either contain the isSpeakerMuted value, incase of a successful toggle or null when the toggling fails
+   * result: True means incoming audio is muted and false means incoming audio is unmuted
+   * error can either contain an error, incase of an error or null when toggle is successful
+   * error: Incase of an error, errorCode will be 500 - internal error
+   */
+  export function toggleIncomingClientAudio(callback: (error: SdkError, result: boolean | null) => void): void {
+    if (!callback) {
+      throw new Error('[toggle incoming client audio] Callback cannot be null');
+    }
+    ensureInitialized();
+    const messageId = sendMessageRequestToParent('toggleIncomingClientAudio');
+    GlobalVars.callbacks[messageId] = (error: SdkError | null, result: boolean | null) => {
+      if (callback) {
```

Comment:
I don't think this check is needed since you already ensure that the callback is defined above.

Reply:
Removed the check